config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: quay.io/rvargasp/opentelemetry-operator
```

Comment:
this needs to be removed

Reply:
It was, but I commited again! :D sorry, reverted